packages/sw-background-sync-queue/test/browser/background-sync-queue.js

```diff
@@ -61,17 +61,17 @@ describe('background sync queue test', () => {
       CALLBACKS);
   });
 
-  it('check push proxy', () => {
+  it('check push proxy', async () => {
     const currentLen = backgroundSyncQueue._queue.queue.length;
-    backgroundSyncQueue.pushIntoQueue({request: new Request('http://lipsum.com')}).then( (e) => {
+    await backgroundSyncQueue.pushIntoQueue({request: new Request('http://lipsum.com')}).then( (e) => {
```

Comment:
`await` is an alternative to attaching a `.then()`.

So this could be:

```js
await backgroundSyncQueue.pushIntoQueue({request: new Request('http://lipsum.com')});
chai.assert(...);
```

That will block until `pushIntoQueue()` completes, and then execute the assertion. Again, it's just an alternative to using `.then()`, but I think it ends up be clearer, especially for cases when you're ignoring the value that the async operation resolves with (like you are in this case).